src/server/ndjsonWorker.js

```diff
@@ -41,8 +45,16 @@ ndjsonWorker.process(async job => {
       const data = JSON.parse(resourceStr);
       return updateResource(data.id, data, data.resourceType);
     });
+  let count = await getCurrentSuccessfulImportCount(clientId);
+  const results = await Promise.all(insertions);
 
-  await Promise.all(insertions);
+  results.map(result => {
+    if (result.created === true) {
+      count = count + 1;
+    }
+  });
+  console.log('the value of count is: ' + count);
+  await updateSuccessfulImportCount(clientId, count);
```

Comment:
This will run into a weird race condition where multiple ndjson workers could be updating the count at one time. 

The strategy we had around this was to use `process.send` as you can see a few lines below this in order to tell the main process to update the db from there. We can add the number of successfully resolved promises to the object in the `process.send` call to pass along the total number of resources to update the count with. Then you can do this database update from there.